docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -612,6 +612,21 @@ iceberg.snowflake-catalog.password=secret
 iceberg.snowflake-catalog.database=db
 ```
 
+When using the Snowflake catalog, data management tasks such as creating tables,
+must be performed in Snowflake because using the catalog from external systems
+like Trino only supports `SELECT` queries as read operations.
```

Comment:
I believe there are more statements. e.g., DESCRIBE, SHOW
No need to mention SELECT here in my opinion.

Reply:
should it be reverted to read-only then?

Reply:
I rejigged to keep the SELECT mention but also link to the read operations section. This way we hint at more but dont claim for all of them to work like in connectors.